src/app/core/services/http-params-mapper.service.ts

```diff
@@ -76,8 +76,13 @@ export class HttpRequestParamsMapper {
       [DataAddressProperty.queryParams]: httpRequestParams.queryParams,
       ...mapKeys(
         httpRequestParams.headers,
-        (k) => `${DataAddressProperty.header}:${k}`,
-      ),
+        (k) => {
+          if(k.toLowerCase() === 'content-type') {
+            return DataAddressProperty.contentType;
```

Comment:
please add a comment to this if, that this is required because the EDC overwrites the ContentType header if you set it as header, because it has this special "field"

Reply:
Resolved in https://github.com/sovity/edc-ui/pull/661/commits/672bae1c49d110ac5b996cd4254a81c52ded5203.